sway/desktop/render.c

```diff
@@ -257,7 +246,6 @@ struct fx_framebuffer *get_main_buffer_blur(struct fx_renderer *renderer, struct
 	pixman_region32_copy(&damage, original_damage);
 	wlr_region_transform(&damage, &damage, transform,
 			monitor_box.width, monitor_box.height);
-	wlr_region_expand(&damage, &damage, get_blur_size());
```

Comment:
region is already being transformed by the full monitor size

Reply:
This should be reverted. The blur damage should be expanded further to reduce the vignette around the blur:
![image](https://user-images.githubusercontent.com/35975961/236669709-69de6fbb-db03-4863-8fc8-3fc50b00e9ff.png)
![image](https://user-images.githubusercontent.com/35975961/236669719-03a0123e-2f8d-4489-8004-3b2e421f10dd.png)